solr/core/src/test/org/apache/solr/cli/ZkSubcommandsTest.java

```diff
@@ -490,7 +523,7 @@ public void testGet() throws Exception {
 
     final String standardOutput2 = byteStream2.toString(StandardCharsets.UTF_8);
     assertTrue(standardOutput2.startsWith("Copying from 'zk:/getNode'"));
-    byte[] fileBytes = Files.readAllBytes(Paths.get(localFile.getAbsolutePath()));
+    byte[] fileBytes = Files.readAllBytes(localFile.toAbsolutePath());
```

Comment:
why call toAbsolutePath?  there's no point; we're going to read it not show it or pass it off somewhere.  I observe many developers love calling getAbsolutePath but I don't get the point.